README.md

```diff
@@ -32,6 +32,11 @@ License: Apache Software License 2.0
 1. Run the server and set it to automatically refresh on changes
    2. `npm install`
    3. `npm run dev`
+1. (Optional) To load data from fixtures, use the following command and replace the fixture_name with fixtures available.
+   1. `python manage.py loaddata fixture_name`
```

Comment:
It might be worth mentioning that this command must be run from the `src` directory or the command must be modified to `python src/manage.py loaddata fixture_name` if in the root directory.